refinery/core/utils.py

```diff
@@ -853,13 +849,14 @@ def get_full_url(relative_url):
         return None
 
     return "{}://{}{}".format(
-        settings.REFINERY_URL_SCHEME, current_site.domain, relative_url
+        settings.REFINERY_URL_SCHEME, current_site.domain, string
     )
 
 
-def is_url(string):
-    """Check if a given string is a URL"""
-    return urlparse(string).scheme != ""
+def is_absolute_url(string):
+    if string and '://' in string:
```

Comment:
- I think you want a stricter regex: I don't think anything would preclude `://` from showing up in the path or query
- just return the result of the expression? `if` is redundant.

Reply:
* Keep in mind that it is not trying to validate a URL. I figured if it's good enough for Django then it's good enough for Refinery: https://github.com/django/django/blob/5cc28dc752c3ae78456bb835c3ba195489fc26d7/django/conf/urls/static.py#L22.
* I'll refactor to simplify.